ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -268,26 +313,46 @@ pub(crate) struct AOTModule {
     types: Vec<Type>,
 }
 
-impl Display for AOTModule {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        writeln!(f, "# IR format version: {}", self.version)?;
-        writeln!(f, "# Num funcs: {}", self.funcs.len())?;
-        writeln!(f, "# Num consts: {}", self.consts.len())?;
-        writeln!(f, "# Num types: {}", self.types.len())?;
+impl AOTModule {
+    /// Compute variable names for instructions that generate a value.
+    fn compute_variable_names(&self) {
+        for f in &self.funcs {
+            for (bb_idx, bb) in f.blocks.iter().enumerate() {
+                for (inst_idx, inst) in bb.instrs.iter().enumerate() {
+                    if inst.opcode.generates_value() {
+                        *inst.name.borrow_mut() = Some(format!("{}_{}", bb_idx, inst_idx));
+                    }
+                }
+            }
+        }
+    }
+
+    // Note that we don't implement `YkDisplay` on the top-level `AOTModule`. Imagine it, it would
```

Comment:
This comment seems to be floating? Is it a doc comment or ...?

Reply:
It's a doc for the two following two methods. I didn't really know how to do it...

Reply:
I totally failed to understand that's what it's referring to...

Reply:
I just killed the comment altogether (in 20b53e05b6db8e1a775d99eb084c9be36505016a). I hope it should be obvious, and we can't explain every little implementation detail.